server/src/main/java/org/candlepin/resource/util/GuestMigration.java

```diff
@@ -86,15 +86,36 @@ public GuestMigration buildMigrationManifest(ConsumerDTO incoming, Consumer exis
             return this;
         }
 
-        Consumer incomingConsumerEntity = new Consumer();
-        List<GuestId> guestIds = new ArrayList<>();
-        for (GuestIdDTO dto : incoming.getGuestIds()) {
-            GuestId entity = new GuestId();
-            GuestIdResource.populateEntity(entity, dto);
-            guestIds.add(entity);
+        manifest = new MigrationManifest(existing);
```

Comment:
This code looks correct enough, but something about this isn't sitting well with me, and I can't quite put my finger on what the issue is. I'll mention this as a post-scrum discussion on Monday and try to figure out where we take this class and the surrounding/provided functionality.